### PR TITLE
[SSL] staging environment beta

### PR DESCRIPTION
**Pull request by kodster28**
Addresses PCX-1141.

Screenshot for ease of review:
image

### CODE REVIEW

File: products/ssl/src/content/edge-certificates/staging-environment.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+### Access
+
+Currently, staging environments are only available to Enterprise customers participating in an open beta. To get access to the beta, contact your Account team.
+
+### Functionality
+
+At the moment, staging environments have limited functionality:
+
+- Only custom (modern) certificates
+- Only accessed via the dashboard
```
Comment (kodster28) [inline products/ssl/src/content/edge-certificates/staging-environment.md:74]: I'm okay with this two-item list b/c we'll likely be adding more soon based on feedback/requests.

File: products/ssl/src/content/edge-certificates/staging-environment.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+Use your certificate staging environment to test new custom (modern) certificates before pushing them to your production environment. This process helps you solve potential certificate problems **before** there's an incident, such as when:
+
+- You make a mistake when uploading a new custom certificate.
+- You misunderstand the order of your certificates.
+- Clients have previously pinned your custom certificate, causing a TLS termination error.
+
+<Aside type="note">
+
+Staging environments are only to Enterprise customers as part of an open beta. For access, reach out to your Account team.
```
Comment (dcpena) [inline products/ssl/src/content/edge-certificates/staging-environment.md]: Looks like you've forgotten a word here. I assume it's "available"?

File: products/ssl/src/content/edge-certificates/staging-environment.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+You should confirm whether:
+
+- TLS termination is successful.
+- The right certificate is being served at the edge.
+- Any clients are pinning the old certificate.
+
+### Step 3 — Push certificate to production
+
+Asssuming there are no issues, you will want to push your custom (modern) certificate to your production environment:
```
Comment (dcpena) [inline products/ssl/src/content/edge-certificates/staging-environment.md]: Seems a little wordy to say "you will want to," and you could just start at "push."

**Review by dcpena — APPROVED** [review]

File: products/ssl/src/content/edge-certificates/staging-environment.md (hunk 2 above)
Comment (kodster28) [reply products/ssl/src/content/edge-certificates/staging-environment.md]:
```suggestion
Staging environments are available only to Enterprise customers as part of an open beta. For access, reach out to your Account team.
```

File: products/ssl/src/content/edge-certificates/staging-environment.md (hunk 3 above)
Comment (kodster28) [reply products/ssl/src/content/edge-certificates/staging-environment.md]:
```suggestion
Asssuming there are no issues, push your custom (modern) certificate to your production environment:
```

**Pull request merged**